.expeditor/config.yml

```diff
@@ -3,67 +3,26 @@
 
 product_key: inspec
 
-rubygems:
- - inspec
- - inspec-core
- - inspec-bin:
-    gemspec_path: ./inspec-bin/
- - inspec-core-bin:
-    gemspec_path: ./inspec-bin/
 
 pipelines:
- - habitat/build:
+ - verify:
+    description: Pull Request validation tests
+    public: false
```

Comment:
Please remove this line, if left blank it assumes private.

Reply:
Done!